docs/source/index.rst

```diff
@@ -33,6 +33,8 @@ The source code is freely available at
 
    faq
 
+   extensions
```

Comment:
Shall we make a second toc tree with caption Development? With the API ref and building extensions? Basic usage of extensions can be described in the other toc tree(?)

Reply:
I agree, see the update!